app/views/plates/show.html.erb

```diff
@@ -1,18 +1,33 @@
+<a href="<%= labware_path(@plate) %>">&lt;&lt; Back to Labware <%= @plate.human_barcode %></a>
 
 <h2>QC Results</h2>
 
-<h3>Concentration</h3>
-<%= render partial: "qc_table", locals: { plate: @plate, qc_type: "concentration" } %>
+<h3>
+Concentration (ng/µl)
+<small class="text-muted">Measured from QC File upload</small>
```

Comment:
I would personally go for less capitalisation....

`Measured from QC file upload`
`Calculated after bed verification`

I think we have a tendency to promote phrases we say a lot ('Bed Verification') to official proper nouns 💪